### ISSUE TITLE
[Bug] When updating Ryujinx, the program closes and doesn't update.

### ISSUE THREAD

**Issue by TalesofTim** [Ryujinx/Ryujinx#7178]
### Description of the issue

When I open the program I sometimes get a little popup saying that there's an update. I was able to update using that until it got to update 1.1.1368 but then it just stopped updating altogether. It will start the update, rename a bunch of files and then close. When I restart the program, it hasn't updated and asks me to update again.

### Reproduction steps

I am honestly not sure, I have changed nothing. All I do is start the program and then click update.

### Log file

```
00:00:00.187 |I| Configuration LogValueChange: BackendThreading set to: Auto
00:00:00.188 |I| Configuration LogValueChange: ResScale set to: 1
00:00:00.197 |I| Configuration LogValueChange: ResScaleCustom set to: 1
00:00:00.197 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.199 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
00:00:00.437 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
00:00:00.437 |I| Configuration LogValueChange: PreferredGpu set to: 
00:00:00.440 |I| Configuration LogValueChange: EnableDockedMode set to: True
00:00:00.447 |I| Configuration LogValueChange: EnableVsync set to: True
00:00:00.447 |I| Configuration LogValueChange: EnableShaderCache set to: True
00:00:00.447 |I| Configuration LogValueChange: EnableTextureRecompression set to: False
00:00:00.447 |I| Configuration LogValueChange: EnableMacroHLE set to: True
00:00:00.447 |I| Configuration LogValueChange: EnableColorSpacePassthrough set to: False
00:00:00.448 |I| Configuration LogValueChange: AntiAliasing set to: None
00:00:00.449 |I| Configuration LogValueChange: ScalingFilter set to: Bilinear
00:00:00.449 |I| Configuration LogValueChange: ScalingFilterLevel set to: 80
00:00:00.449 |I| Configuration LogValueChange: EnablePtc set to: True
00:00:00.449 |I| Configuration LogValueChange: EnableInternetAccess set to: False
00:00:00.449 |I| Configuration LogValueChange: EnableFsIntegrityChecks set to: True
00:00:00.449 |I| Configuration LogValueChange: FsGlobalAccessLogMode set to: 0
00:00:00.450 |I| Configuration LogValueChange: AudioBackend set to: SDL2
00:00:00.450 |I| Configuration LogValueChange: AudioVolume set to: 1
00:00:00.453 |I| Configuration LogValueChange: MemoryManagerMode set to: HostMappedUnsafe
00:00:00.453 |I| Configuration LogValueChange: ExpandRam set to: False
00:00:00.453 |I| Configuration LogValueChange: IgnoreMissingServices set to: False
00:00:00.453 |I| Configuration LogValueChange: UseHypervisor set to: True
00:00:00.454 |I| Configuration LogValueChange: MultiplayerMode set to: Disabled
00:00:00.678 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.1366
00:00:00.685 |N| Application Print: Operating System: Microsoft Windows 10.0.19045 (X64)
00:00:00.685 |N| Application Print: CPU: Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz ; 8 logical
00:00:00.689 |N| Application Print: RAM: Total 16 GiB ; Available 6.6 GiB
00:00:00.692 |N| Application PrintSystemInfo: Logs Enabled: Info, Warning, Error, Guest, Stub
00:00:00.693 |N| Application PrintSystemInfo: Launch Mode: UserProfile
```

### OS

Windows 10

### Ryujinx version

1.1.1366

### CPU

i7-7700

### GPU

NVIDIA geForce GTX 1070

### RAM

16 GB

### Additional context?

I was running this on 1.1.1368 because it did update once before it went bad. But since then I haven't been able to update even if I rolled the program back to 1.1.1366.

Also, the Log isn't anything recent because it didn't have anything recent.

**Comment by TalesofTim** [comment]
After posting this, I went into the program to figure out what was going on. Turns out that I was running Ryujinx.exe and not Ryujinx.Ava.exe because running the Ava one actually does show that it's updated. Which is weird.

**Comment by TalesofTim** [comment]
But running that one makes playing totk, which ran fine before, run slower and with visual bugs.

**Comment by TalesofTim** [comment]
Okay, so I fixed the issue and was able to update. Here is how I solved it ... 

I noticed that there was still a file called runjinxold or something like that. But when I tried to delete it I got an error that said ryujinx was still using that file even when ryujinx wasn't running. So I restarted my computer, then deleted the file. Then I reinstalled the entire program and it worked. It does crash now every once in a while which it didn't do before, but it doesn't do that too often. So I'm marking this as closed.
